forge-gui/res/cardsfolder/r/ria_ivor_bane_of_bladehold.txt

```diff
@@ -4,7 +4,7 @@ Types:Legendary Creature Phyrexian Knight
 PT:3/4
 K:Battle cry
 T:Mode$ Phase | Phase$ BeginCombat | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigEffect | TriggerDescription$ At the beginning of combat on your turn, the next time target creature would deal combat damage to one or more players this combat, prevent that damage. If damage is prevented this way, create that many 1/1 colorless Phyrexian Mite artifact creature tokens with toxic 1 and "This creature can't block."
-SVar:TrigEffect:DB$ Effect | ValidTgts$ Creature | Duration$ UntilEndOfCombat | ReplacementEffects$ StrikeWithAwe | ExileOnMoved$ Battlefield | RememberObjects$ Targeted | AILogic$ Fog
+SVar:TrigEffect:DB$ Effect | ValidTgts$ Creature | IsCurse$ True | Duration$ UntilEndOfCombat | ReplacementEffects$ StrikeWithAwe | ExileOnMoved$ Battlefield | RememberObjects$ Targeted | AILogic$ Fog
```

Comment:
The AI Api for this effect neither supports IsCurse or that AILogic

they should either be removed and the card marked as unplayable or support be added

Reply:
RemoveDeck:All instead since this card doesn't make sense without the trigger